include/plog/Appenders/AndroidAppender.h

```diff
@@ -12,6 +14,7 @@ namespace plog
         {
         }
 
+#ifdef ANDROID
```

Comment:
I'm not sure it's a good idea to provide an empty implementation of AndroidAppender. It could conceal errors. What do you think?